bots/discord/OnMessageCreate.go

```diff
@@ -80,4 +81,62 @@ func OnMessageCreate(s *discordgo.Session, msg *discordgo.MessageCreate) {
 		s.ChannelMessageSend(msg.ChannelID, msg.Author.Mention()+" B-baaaaaaaka! Y..you...you want to...TOUCH MY CODE?!\n\nhttps://github.com/animenotifier/notify.moe/tree/go/bots/discord")
 		return
 	}
+
+	// Set the specific region role for the user
+	if strings.HasPrefix(msg.Content, "!region ") {
+
+		regions := map[string]string{
```

Comment:
This doesn't need to be allocated every time the region command is run.
Use a global variable outside the function, it's not bad in this case.